helm/templates/cloudwatch-agent-daemonset.yaml

```diff
@@ -26,6 +26,8 @@ metadata:
 spec:
   image: {{ template "cloudwatch-agent.image" . }}
   mode: daemonset
+  nodeSelector:
+    kubernetes.io/os: linux
```

Comment:
please add the same `nodeSelector` block to `dcgm-exporter-daemonset` spec

Reply:
Addressed. Added tag in dcgm exporter daemonset.